docs/Hosts/Spawn-Hosts.md

```diff
@@ -55,6 +55,149 @@ lifetime up to 30 days past host creation.
+
+![Stop the host and keep off](../images/stop_host_keep_off.png)
+
+This can also be done through the CLI:
+
+```sh
+evergreen host stop --host <host_id> --keep-off
+```
+
+#### Temporary Exemptions
+
+If you need your host to temporarily ignore its sleep schedule, you can request a temporary exemption for your host.
+During a temporary exemption, the sleep schedule will not take effect at all, so Evergreen will not stop/start your host
+unless you stop/start it manually. This is useful if you have a one-off need to keep your host on without interruption.
+For example, if you're running a test overnight on the host and you don't want the host to be stopped, you can request a
+temporary exemption until tomorrow to keep the host on. Another example is if it's outside your working hours (and the
+host has already automatically stopped for the night for its sleep schedule) but you want to check a file in your host,
+you can request a temporary exemption and then turn your host on. The sleep schedule will let your host stay on, and
+will ignore the host's sleep schedule until the temporary exemption ends.
+
+You can set a temporary exemption in the Spruce edit host modal:
+
+![Temporary exemption](../images/sleep_schedule_temporary_exemption.png)
+
+Alternatively, you can set a temporary exemption from the CLI:
+
+```sh
+evergreen host modify --host <HOST_ID> --extend-temporary-exemption <NUM_HOURS>
+```
+
+You can temporarily exempt your host from the sleep schedule for up to a month from today.
+
+#### Requesting a Permanent Exemption
+
+If for some reason your host cannot use a sleep schedule at all, you can request that your host be permanently exempt
+from the sleep schedule feature. If your host is permanently exempt, it won't have any sleep schedule set and Evergreen
+will not stop/start your host according to a recurring sleep schedule. A permanent exemption from setting a sleep
+schedule would allow your host to be kept on 24/7.
+
+**Requesting a permanent exemption is not recommended unless you have set up [the auto-sleep
+script](#host-auto-sleep-script) (and want it to substitute for a recurring sleep schedule) or you have a particular
+reason that your host must be kept on at all times.** This is because keeping hosts on 24/7 is not an efficient use of
+AWS resources since the host incurs costs to run while it's not being actively used. In addition, users who only
+occasionally need their host to be kept on for an extended period of time can set [temporary
+exemptions](#temporary-exemptions) when needed.
+
+If you'd like to request a permanent exemption, please file a DEVPROD ticket with Evergreen App as the Dev Prod service
+and use the title "Permanent Exemption Request". In it, please include your host ID and an explanation of why you'd like
+your host to be permanently exempt from the sleep schedule (and if relevant, why alternative options like temporary
+exemptions would not be sufficient for your usage).
```

Comment:
Can we add a sentence here directing users to `##### Requesting a Permanent Exemption with the Auto-Sleep Script `to follow the instructions there if that's the reason they are filling the ticket for an exemption?

Reply:
Added a link.